### ISSUE TITLE
Extension does not show 'arguments are not aligned' errors

### ISSUE THREAD

**Issue by Junkern** [Microsoft/vscode-tslint#234]
I have installed the `tslint` extension. It shows no linting errors for my ts files. I am using https://github.com/t-sauer/mocha-tslint to verify the linting rules with mocha (as part of my CI). It fails, however: `arguments are not aligned at line 88, character 10`. I know the reason (https://github.com/palantir/tslint/issues/742) but I can't figure out why the tslint extension does not show those errors. I have a `tslint.json`:

```json
{
  "extends": "tslint-config-airbnb",
  "rules": {
    "semicolon": [
      true,
      "never"
    ],
    "trailing-comma": [
      false
    ],
    "indent": [
      true,
      "spaces",
      2
    ],
    "no-trailing-whitespace": true,
    "max-line-length": [
      false
    ],
    "no-use-before-declare": true,
    "no-unused-variable": true
  }
}
```

Can it be due to local and global tslint-installations/configs interfering or something like that?

**Comment by egamma** [comment]
Can you verify that tslint is running? When tslint is running you see a status bar item 'tslint' on the right of the status bar. Does it show up? When you click the item what is shown in the output panel?

Can you run tslint on the file with this failure from the command line? If it succeeds how does the command line look like?

**Comment by Junkern** [comment]
Status bar shows tslint. Output is:
```
[Info  - 11:47:36 AM] Linter is running.
Warning: The 'no-use-before-declare' rule requires type checking
Warning: The 'no-unused-variable' rule requires type checking
```

Running tslint from the command line results in the same output, whereas running the `mocha-tslint` throws the error described in the first post.

**Comment by egamma** [comment]
>Running tslint from the command line results in the same output, whereas running the  mocha-tslint  throws the error described in the first post.

Then you need to find what is different between the command line and mocha-tslint setup. Is the rule that fails a tslint core rule? What is the rule name? The tslint-config-airbnb defines additional rules and I suspect that in the mocha-tslint setup you define a rules directory so that this additional rules are found.

**Comment by Junkern** [comment]
The rule is specified in the `tslint-config-airbnb` (https://github.com/progre/tslint-config-airbnb/blob/master/tslint.js) and is a core tslint rule: https://palantir.github.io/tslint/rules/align/

I took a look at the `mocha-tslint` code (https://github.com/t-sauer/mocha-tslint/blob/master/index.js), and all it basically does is:
```
const TSLint = require('tslint');
const Linter = TSLint.Linter;
const linter = new Linter({
  fix: false,
  formatter: 'json'
});
const source = sourceBuffer.toString();
linter.lint(file, source.toString(), config);
const result = linter.getResult();
```

The `config` is the `tslint.json` file.

Even adding `"align": [true, "arguments", "parameters"]` directly to my `tslint.json` does not show any errors

**Comment by Junkern** [comment]
Ok, I updated. My package.json now contains tslint in version `5.5.0`. When I execute tslint directly on the file (`node_modules/tslint/bin/tslint -c ./tslint.json ./test/index.ts `), the `arguments are not aligned` shows up. After a visual studio code restart, it also showd up in VSCode. Thanks for the nudges in the right direction;)
